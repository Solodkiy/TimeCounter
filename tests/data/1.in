14,00 - 15,45
16.08 - 16.58