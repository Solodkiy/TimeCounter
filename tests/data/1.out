14:00 - 15:45 = 1:45
16:08 - 16:58 = 0:50
                ----
                2:35